### PR TITLE
Avoid extra console write for clearing on Unix

### PR DESCRIPTION
**Pull request by stephentoub**
https://github.com/dotnet/runtime/pull/88487#discussion_r1265344770

### CODE REVIEW

**Review by lewing — APPROVED** [review]

File: src/libraries/System.Console/tests/TermInfo.Unix.cs [hunk 1]
```diff
@@ -103,20 +103,6 @@ public void TermInfoVerification(string termToTest, string expectedForeground, s
...
-    [Fact]
-    [PlatformSpecific(TestPlatforms.AnyUnix)] // Tests TermInfo
-    public void TermInfoE3CanBeRead()
-    {
-        bool ScrollbackBufferIsSet(TerminalData data) => !string.IsNullOrEmpty(data.TerminalDb.ClearScrollbackBuffer);
-
-        // XTerm defines E3 for clearing scrollback buffer and tmux does not.
-        // This can't be added to TermInfoVerification because xterm-256color sometimes has E3 defined (e.g. on Ubuntu but not macOS)
-        XTermData xTermData = new();
-        Assert.True(ScrollbackBufferIsSet(xTermData));
```
Comment (adamsitnik) [inline src/libraries/System.Console/tests/TermInfo.Unix.cs:115]: I don't like the removal of unit test. I would prefer to change the test to verify that `data.TerminalDb.Clear` is given string for these two terminals and keep the test coverage.

**Review by adamsitnik — COMMENTED** [review]
Thank you for the optimization @stephentoub. PTAL at my comment.

File: src/libraries/System.Console/tests/TermInfo.Unix.cs (hunk 1 above)
Comment (stephentoub) [reply src/libraries/System.Console/tests/TermInfo.Unix.cs:115]: The unit test isn't really validating anything meaningful, but I can put it back.

Comment (adamsitnik) [reply src/libraries/System.Console/tests/TermInfo.Unix.cs:115]: Please put it back.

Comment (adamsitnik) [reply src/libraries/System.Console/tests/TermInfo.Unix.cs:115]: @stephentoub ping (it will become a stale PR soon)

Comment (stephentoub) [reply src/libraries/System.Console/tests/TermInfo.Unix.cs:115]: Please feel free to add a test to the PR or merge it as is. I've had other priorities. Thanks.

**Review by adamsitnik — APPROVED** [review]
LGTM, thank you @stephentoub !

**Pull request merged**